web/src/services/theme/publishedAppProvider.tsx

```diff
@@ -1,15 +1,14 @@
 import { ThemeProvider } from "@emotion/react";
 import { ReactNode } from "react";
 
-import classicTheme from "@reearth/classic/theme/reearthTheme/darkTheme";
-
 import GlobalStyle from "./reearthTheme/common/globalStyles";
-import betaTheme from "./reearthTheme/darkTheme";
+import darkTheme from "./reearthTheme/darkTheme";
+import lightTheme from "./reearthTheme/lightTheme";
 
 const Provider: React.FC<{ children?: ReactNode }> = ({ children }) => {
   const theme = {
-    classic: classicTheme,
-    ...betaTheme,
+    ...lightTheme,
+    ...darkTheme,
```

Comment:
```suggestion
    ...darkTheme,
```
Seems we only use darkTheme here (i don't know the reason though). Let's keep the behavior as-is for this PR.

Reply:
Yes, the emotion theme provider gave an type error if I tried to specify properties to it like in the screenshot. I think we will need to refactor this when we decide how to add styles. I think TailwindCSS would be a good one for us.
<img width="1008" alt="Screenshot 2024-07-17 at 12 15 52 PM" src="https://github.com/user-attachments/assets/853454d5-bfc2-4991-9208-965a051841cf">

Reply:
For published page the theme here might not be in actual use. Just give it darkTheme as we did before is okey i think. What you did on screen shot changed the type and it will lead to type error for sure.
Tailwind & styled component both have their own pros and cons, both popular anyway. Maybe we will change our choice when necessary but that's another topic.

Reply:
Got it. I misunderstood. I've removed the light theme and kept dark theme.